llvm/lib/Target/STM8/STM8ISelLowering.cpp

```diff
@@ -22,7 +22,7 @@ namespace llvm {
 
 STM8TargetLowering::STM8TargetLowering(const STM8TargetMachine &TM,
                                        const STM8Subtarget &STI)
```

Comment:
If `STI` is not used anywhere, compilers will usually complain about it. To work around it, give it no name:
```suggestion
                                       const STM8Subtarget &)
```

Reply:
But it is not used rn. It might be in the future